pkgs/games/arx-libertatis/default.nix

```
@@ -85,11 +85,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = {
-    description = ''
-      A cross-platform, open source port of Arx Fatalis, a 2002
-      first-person role-playing game / dungeon crawler
-      developed by Arkane Studios.
-    '';
+    description = "Cross-platform, open source port of Arx Fatalis, a 2002 first-person role-playing game / dungeon crawler developed by Arkane Studios";
```

Probably split into a long description, if we want to keep the extra detail

```suggestion
    description = "First-person role-playing game / dungeon crawler";
```